sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -291,8 +291,12 @@ def GetHeaders(  # pylint: disable=too-many-statements,too-many-branches
             if_none_match_value = options["continuation"]
         elif options.get("isStartFromBeginning") and not options["isStartFromBeginning"]:
             if_none_match_value = "*"
+        elif options.get("startTime"):
+            dt = options.get("startTime")
+            headers[http_constants.HttpHeaders.IfModified_since] = dt.strftime('%a, %d %b %Y %H:%M:%S GMT')
```

Comment:
`dt` could be in any timezone, instead it should be moved to UTC with `astimezone`